tests/feedback.test.ts

```diff
@@ -118,8 +118,8 @@ describe('feedback submission', () => {
     const submittedFeedbackResponse = await feedbackController.submitFeedback({ feedback }, user);
 
     const status = FeedbackStatus.ACKNOWLEDGED;
-    const uuid = submittedFeedbackResponse.feedback;
-    const acknowledgedFeedback = await feedbackController.updateFeedbackStatus(uuid, { status }, admin);
+    const { uuid } = submittedFeedbackResponse.feedback;
+    const acknowledgedFeedback = await feedbackController.updateFeedbackStatus({ uuid }, { status }, admin);
```

Comment:
I don't see any changes to the feedback controller so why this change?

Reply:
I believe it was a missing change from [this PR](https://github.com/acmucsd/membership-portal/pull/173) that wasn't causing the test to fail for some reason.